ansible_ai_connect/ai/api/views.py

```diff
@@ -176,9 +176,18 @@
 
 
 class AACSAPIView(APIView):
+    def __init__(self, **kwargs):
+        super().__init__(kwargs)
+        self.event = None
+        self.model_id = None
+        self.exception = None
+
     def initialize_request(self, request, *args, **kwargs):
+        # Call super first to ensure request object is correctly
```

Comment:
There was a problem with the tests and use of `force_authenticate`. Django had not setup the _forced_ user when we were calling `self.event.set_request` and hence the event contained an `AnonymousUser`.